ui/src/pages/namespace/Explorer/Page/poc/schema/primitives/keyValue.ts

```diff
@@ -1,6 +1,12 @@
 import { TemplateString } from "./templateString";
 import { z } from "zod";
 
+/**
+ * A key-value pair where the key is a simple string and the value is a
+ * template string that can contain dynamic placeholders. This is commonly
+ * used for headers, query parameters, and other simple data structures
+ * that need to support dynamic content from type string.
```

Comment:
```suggestion
 * that need to support strings containing dynamic values.
```
Or just replace "from" with "of".